bin/cli.js

```diff
@@ -224,15 +226,15 @@ localesPaths.forEach(localesPath => {
               //const contextComment = '';
               switch(format) {
                 case 'js':
-                newLocaleContents = `${exportPrefix} ${formatJS(newLocaleContents)}`;
+                newLocaleContents = `${exportPrefix} ${formatJS(newLocaleContents)};`;
```

Comment:
This, and the subsequent `;`, are just so the generate `js` stuff passes our Brightspace linting. Not sure if you have a preference for tailing `;` or not. I could also mitigate this with `eslint` settings specific to our `lang` folder as long as the code above to ignore `.` style "hidden" files goes in.

Reply:
No, this makes sense. I was going to say before that it would break or at least not be valid JSON with the previous change.

Reply:
Yeah I realized my previous change was wrong 😞. Glad it makes sense to add it.